csharp/src/Ice/Communicator.cs

```diff
@@ -572,14 +572,9 @@ private class ObserverUpdater : Instrumentation.IObserverUpdater
 
                 NetworkProxy = CreateNetworkProxy(IPVersion);
 
-                AddEndpointFactory(new TcpEndpointFactory(
-                    new TransportInstance(this, EndpointType.TCP, "tcp", false)));
-                AddEndpointFactory(new UdpEndpointFactory(
-                    new TransportInstance(this, EndpointType.UDP, "udp", false)));
-                AddEndpointFactory(new WSEndpointFactory(
-                    new TransportInstance(this, EndpointType.WS, "ws", false), EndpointType.TCP));
-                AddEndpointFactory(new WSEndpointFactory(
-                    new TransportInstance(this, EndpointType.WSS, "wss", true), EndpointType.SSL));
+                IceAddEndpointFactory(new TcpEndpointFactory(this));
+                IceAddEndpointFactory(new UdpEndpointFactory(this));
+                IceAddEndpointFactory(new WSEndpointFactory(this));
```

Comment:
And presumably the ssl and wss endpoint factories are added when the IceSSL plugin is loaded?

A follow-up PR should further integrate IceSSL in the core:
 - load ssl/wss immediately
 - eliminate the IceSSL prefix in configuration

Reply:
Yes currently ssl/wss are loaded when IceSSL plug-in is initialized, I assume we will also get rid of IceSSL plug-in then